frontend/app/src/data/psl.ts

```diff
@@ -0,0 +1,9380 @@
+/* eslint-disable max-lines */
+
+// Public suffixes list
+// Source: https://github.com/lupomontero/psl/blob/main/data/rules.json
+export const pslSuffixes: string[] = [
```

Comment:
Any reason why we can't use json and import from json?

Reply:
wait this should not be in this PR 😛